massa-client/src/settings.rs

```diff
@@ -26,17 +26,26 @@ pub struct DefaultNode {
     pub public_port: u16,
 }
 
-/// Http Client settings.
-/// the Http Client settings
+/// Client settings
+/// the client settings.
 #[derive(Debug, Deserialize, Clone)]
-pub struct HttpSettings {
+pub struct ClientSettings {
     pub max_request_body_size: u32,
     pub request_timeout: MassaTime,
     pub max_concurrent_requests: usize,
     pub certificate_store: String,
     pub id_kind: String,
     pub max_log_length: u32,
     pub headers: Vec<(String, String)>,
+    pub http: HttpSettings,
+}
+
+///TODO add WebSocket to CLI
```

Comment:
I don't think we want to do this. There is no use case to have a WS in a CLI and there is no CLI format for that

Reply:
May be in headless CLI(non interactive) it could be integrated, but It's not a common usecase so, it's not urgent at all